pkg/flashdb/include/fal_cfg.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_flashdb
+ * @{
+ *
+ * @file
+ * @brief       Flash Abstraction Layer partition configuration
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#ifndef FAL_CFG_H
+#define FAL_CFG_H
+
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Partition table is defined at compile time (not read from flash)
+ */
+#define FAL_PART_HAS_TABLE_CFG
+
+/**
+ * @brief FAL <-> MTD adapter
+ */
+extern struct fal_flash_dev mtd_flash0;
```

Comment:
flashDB boasts that it can do multiple databases on the same system; can we?

It's OK if there is a choice that we'll have a single system-wide KVDB in place, but then the rationale for that should be outlined, and maybe the database initialization should be outside the user application (because then the OS provides "the single" KVDB as a readily usable service).

Reply:
Yea that's an unfortunate consequence of the MTD <-> FAL adaption layer. FAL only expects a single device of an implementation to exist, there is only `write(offset, data, len)` with no way to specify a context pointer to discriminate between different devices. (https://github.com/armink/FlashDB/issues/132#issuecomment-1033138711)

A possible solution would be mapping different MTD backends to different virtual addresses, but I didn't want to go down that rabbit hole.